pandas/tests/io/parser/test_network.py

```diff
@@ -4,21 +4,61 @@
 Tests parsers ability to read and parse non-local files
 and hence require a network connection to be read.
 """
-
 import os
+
 import pytest
+import six
+from moto import mock_s3
```

Comment:
This import may have to be inside the fixture, so that the test module can be imported without moto being installed.